src/components/badge/index.tsx

```diff
@@ -121,7 +121,8 @@ class Badge extends PureComponent<BadgeProps> {
   }
 
   get size() {
-    return this.props.size || 20;
+    const {size, label} = this.props;
+    return label === undefined ? 10 : size !== undefined ? size : 20;
```

Comment:
You can make this last line more readable by setting a default value to size when destructing it

Reply:
That's an improvement and way more readable
Just double checking on what I wrote, can't we set the default size (`DEFAULT_BADGE_SIZE`)  when destructing the size prop?

Reply:
👍 right